docs/pages/upgrading/client-tools-autoupdate.mdx

```diff
@@ -64,6 +64,10 @@ Teleport v17.0.5 git:v17.0.5-0-g7cc4c2a go1.23.4
 
 ### Using `tctl`
 
+<Admonition type="note">
+    `tctl autoupdate` commands are available in versions: >=18.0.0-dev, >=17.2.0
```

Comment:
Is `18.0.0-dev` a Teleport release that users can install? It might make sense to include only `>=17.2.0`.

Reply:
Was not sure about it, so added to get some opinions because we publish nightly builds with `18.0.0-dev` version